session.go

```diff
@@ -391,3 +407,40 @@ func heartbeat(txn *spanner.ReadWriteStmtBasedTransaction, priority pb.RequestOp
 	_, err := iter.Next()
 	return err
 }
+
+func parseDirectedReadOption(directedReadOptionText string) (*pb.DirectedReadOptions, error) {
+	var replicaSelections []*pb.DirectedReadOptions_ReplicaSelection
+
+	directedReadOption := strings.Split(directedReadOptionText, ":")
+	if len(directedReadOption) > 2 {
+		return nil, fmt.Errorf("directed read option must be in the form of <replica_region>:<replica_type>, but got %q", directedReadOptionText)
```

Comment:
Maybe `<replica_location>` would be better since API uses `location`: https://cloud.google.com/spanner/docs/directed-reads#parameters

Reply:
I've unified the inconsistent.